### PR TITLE
First pass Directory Rename

### PR DESCRIPTION
**Pull request by enuessle**

### CODE REVIEW

**Comment by enuessle** [comment]
Also closes #568

File: src/client/useFileCRUD.ts [hunk 1]
```diff
@@ -75,20 +75,30 @@ export const useFileCRUD = ({
       oldName: string,
       newName: string,
     ) => {
+      console.log(path);
```
Comment (curran) [inline src/client/useFileCRUD.ts:78]:
```suggestion
```

File: src/client/useFileCRUD.ts [hunk 2]
```diff
@@ -75,20 +75,30 @@ export const useFileCRUD = ({
...
+      console.log(oldName);
```
Comment (curran) [inline src/client/useFileCRUD.ts:79]:
```suggestion
```

File: src/server/index.js [hunk 3]
```diff
@@ -158,6 +222,23 @@ const save = () => {
       fs.writeFileSync(current.name, current.text);
     }
   }
+  // TODO deleted all directories under directoriesToDelete
+
+  for(const dir of directoriesToDelete){
```
Comment (curran) [inline src/server/index.js:227]: Excellent!

**Review by curran — APPROVED** [review]
This looks great! Thank you so much.

**Pull request merged**